libcontainer/integration/execin_test.go

```diff
@@ -291,46 +291,55 @@ func TestExecInTTY(t *testing.T) {
 		Args: []string{"ps"},
 		Env:  standardEnvironment,
 	}
-	parent, child, err := utils.NewSockPair("console")
-	if err != nil {
-		ok(t, err)
-	}
-	defer parent.Close()
-	defer child.Close()
-	ps.ConsoleSocket = child
-	done := make(chan (error))
-	go func() {
-		f, err := utils.RecvFd(parent)
-		if err != nil {
-			done <- fmt.Errorf("RecvFd: %w", err)
-			return
-		}
-		c, err := console.ConsoleFromFile(f)
+
+	// Repeat to increase chances to catch a race; see
+	// https://github.com/opencontainers/runc/issues/2425.
+	for i := 0; i < 300; i++ {
```

Comment:
Now, this commit is totally optional -- I am not sure myself I want it, since the bug is fixed and it takes about 5s on my laptop for this test now.